symfony/.env.test

```diff
@@ -0,0 +1,10 @@
+# define your env variables for the test env here
```

Comment:
should that file be committed? if necessary I'd rather have it named `.env.dist`

Reply:
Well, by default the recipes ships this file ( what people actually get in symfony ). So yeah - I think we can keep it.

Reply:
.env.test != .env.dist

.env.test = environments that should be used when running phpunit
.env.dist = the example file that you should rename to `.env`.

Reply:
good for me!